forge/info.rkt

```diff
@@ -2,7 +2,7 @@
 
 (define collection "forge")
 
-(define version "1.5.0")
+(define version "1.7.0")
```

Comment:
Why not 1.6.0 ?
cc @tnelson

Reply:
This change seems to be coming from the main branch -- dev seems behind

Reply:
This gets automatically incremented on the master branch by a github action. We just need to remember to merge in that increment from master->dev whenever doing a public release.